src/program/lwaftr/run/run.lua

```diff
@@ -85,13 +100,17 @@ function parse_args(args)
    lib.dogetopt(args, handlers, "b:c:n:m:vD:hir:",
       { conf = "c", ["v4-pci"] = "n", ["v6-pci"] = "m",
         verbose = "v", duration = "D", help = "h",
-        virtio = "i", ["ring-buffer-size"] = "r" })
+        virtio = "i", ["ring-buffer-size"] = "r", cpu = 1,
```

Comment:
This means that the --cpu option takes one argument.  This is the way to specify long options without short counterparts.  (I would use -c but it's taken already, and I don't want to use a different option because that would be confusing relative to taskset).